pkg/controller/statusmanager/pod_status.go

```diff
@@ -19,6 +19,7 @@ import (
 
 	"github.com/vmware/nsx-container-plugin-operator/pkg/controller/sharedinfo"
 	operatortypes "github.com/vmware/nsx-container-plugin-operator/pkg/types"
+	// "github.com/operator-framework/operator-sdk/pkg/log/zap"
```

Comment:
if we don't need this anymore, can we remove the line?

Reply:
ops, it's a debug code for loglevel, forgot it... removed, thanks!